common-practices-tools/contribution/contrib-first.md

```diff
@@ -0,0 +1,24 @@
+# Contrib First
+
+Whenever we are building something that could be of use to more than just one project or client, we build it as contributed work first.
+
+## Rationale for contrib first
+
+-   **Fiscal responsibility** - Building it and contributing it means that other government agencies will never have to pay to build the same thing. This helps agencies comply with Federal Source Code Policy: Achieving Efficiency, Transparency, and Innovation through Reusable and Open Source Software [OMB Memorandum M-16-21](https://obamawhitehouse.archives.gov/sites/default/files/omb/memoranda/2016/m_16_21.pdf)
+-   **Reusablity** - CivcActions other clients can benefit from work that was already done.
+-   **Security** - contributing our work to an open source project like Drupal means it it receives security coverage by the Drupal security team and the public.
```

Comment:
```suggestion
-   **Security** - contributing our work to an open source project like Drupal means it may receive security coverage by the Drupal security team and the public.
```